recipes/label_parsing.ipynb

```diff
@@ -0,0 +1,3063 @@
+{
```

Comment:
<p>"Children jobs" -&gt; "Child jobs"</p><p> </p><p>"For children jobs, the <code>.children</code> attribute allows to access the children labels" -&gt; "Use the <code>.children</code> attribute to access nested (child) labels"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='142'/>